Dockerfile

```diff
@@ -15,5 +15,8 @@ RUN apk --no-cache add --no-check-certificate ca-certificates \
 
 COPY --from=build /app/main /ryd-proxy
 
+RUN apk add dumb-init
```

Comment:
This sbould be done in line 12 where we are also calling `apk add`, because otherwise docker creates an additional layer for that step which increases thr imahe size

Reply:
Done :+1: